storyblok/storyblok-design-system
# task(INT-408):  Translate sbPagination and SbUploadDialog

In this PR we created the struct to translate the Design System components.
The first translated component was SbPagination.
## Pull request type

Jira Link: [INT-408 - [V2][DS] - Pagination: translate Items per page, page and tooltips](https://storyblok.atlassian.net/browse/INT-408)




## How to test this PR
Go in: https://storyblok-design-system-git-feat-int-408-t-6cf4fb-storyblok-com.vercel.app/?path=/story/design-system-components-sbpagination--default
Change Locale control and check if texts translated.

Go in: https://storyblok-design-system-git-feat-int-408-t-6cf4fb-storyblok-com.vercel.app/?path=/story/design-system-components-sbuploaddialog--default
Change Locale control and check if texts translated.
